core/http.go

```diff
@@ -0,0 +1,6 @@
+package core
+
+// IsStatusCodeIs2xx returns true if the provided status code is not 2xx
+func IsStatusCodeIs2xx(statusCode int) bool {
```

Comment:
Name does not match description. Also, perhaps there's a typo in the name?

Reply:
refactored